content/530.md

```diff
@@ -0,0 +1,20 @@
+## Description
+
+<aside class="warning"><strong>Unofficial:</strong> This HTTP status code is not specified by any RFC.</aside><br>
+
+The HTTP response status code `530 Site Frozen` is an unofficial HTTP status code that is specific to Cloudflare and Pantheon.
+
+#### Cloudflare
+
+Cloudflare uses the `530` status code when an issue occurs between a user and a website server. Accompanied by a more detailed HTTP status code 1XXX, the error is dependent on the specific situation.
+
+#### Pantheon
+
+Pantheon returns the `530 Site Frozen` status code when an inactive Sandbox Site denies HTTP requests. This can be resolved by reactivating the site in the Pantheon User Dashboard.
+
+### See Also
+- [Spec](https://www.rfc-editor.org/rfc/rfc7231.html#status.5xx)
+- [HTTP request methods](https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods)
```

Comment:
I would remove these two links. We state that the code is unofficial and doesn't have an RFC. Listing here an RFC may cause confusion. The MDN one is too generic for this one.